hw/i386/Kconfig

```diff
@@ -99,3 +99,8 @@ config VMPORT
 config VMMOUSE
     bool
     depends on VMPORT
+
+config XBOX
+    select PC
```

Comment:
Probably incomplete list, but `USB_OHCI` and `AC97` are required for us when building (`--without-default-devices` disables them otherwise). Leaving as-is for now.

Reply:
Is this already necessary or just preparation for when we disable those things mentioned in [your other comment](https://github.com/xqemu/xqemu/pull/267/files#r349904622)?

Reply:
Yes this is necessary